src/backends/interflop-bitmask/Makefile.am

```diff
@@ -4,6 +4,7 @@ libinterflop_bitmask_la_CFLAGS = -DBACKEND_HEADER="interflop_bitmask"
 if WALL_CFLAGS
 libinterflop_bitmask_la_CFLAGS += -Wall -Wextra
 endif
+libinterflop_bitmask_la_CFLAGS += -march=native
```

Comment:
I need the flag `-march=native` for all backends which use vector interface else it will not work because the compiler can't match vector types with expected vector set instruction.
Then I added it to all backend because if someone want to vectored is backend it might be not know that.